### PR TITLE
update datepicker.js

### PR DESCRIPTION
**Pull request by napoleonjk**
add international support for the following buttons :Today,Clear,Close

### CODE REVIEW

File: src/datepicker/datepicker.js [hunk 1]
```diff
@@ -460,8 +460,8 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
   showButtonBar: true
 })
 
-.directive('datepickerPopup', ['$compile', '$parse', '$document', '$position', 'dateFilter', 'dateParser', 'datepickerPopupConfig',
-function ($compile, $parse, $document, $position, dateFilter, dateParser, datepickerPopupConfig) {
+.directive('datepickerPopup', ['$locale','$compile', '$parse', '$document', '$position', 'dateFilter', 'dateParser', 'datepickerPopupConfig',
```
Comment (wesleycho) [inline src/datepicker/datepicker.js]: Add a space after `,`

File: src/datepicker/datepicker.js [hunk 2]
```diff
@@ -460,8 +460,8 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
...
+function ($locale,$compile, $parse, $document, $position, dateFilter, dateParser, datepickerPopupConfig) {
```
Comment (wesleycho) [inline src/datepicker/datepicker.js]: Add a space after `,`

File: src/datepicker/datepicker.js [hunk 3]
```diff
@@ -481,6 +481,9 @@ function ($compile, $parse, $document, $position, dateFilter, dateParser, datepi
       scope.showButtonBar = angular.isDefined(attrs.showButtonBar) ? scope.$parent.$eval(attrs.showButtonBar) : datepickerPopupConfig.showButtonBar;
 
       scope.getText = function( key ) {
+        scope['currentText']=$locale.DATETIME_FORMATS.TODAY||scope['currentText'];
```
Comment (wesleycho) [inline src/datepicker/datepicker.js]: Rewrite as `scope.currentText = $locale.DATETIME_FORMATS.TODAY || scope.currentText`

File: src/datepicker/datepicker.js [hunk 4]
```diff
@@ -481,6 +481,9 @@ function ($compile, $parse, $document, $position, dateFilter, dateParser, datepi
...
+	scope['clearText']=$locale.DATETIME_FORMATS.CLEAR||scope['clearText'];
+	scope['closeText']=$locale.DATETIME_FORMATS.CLOSE||scope['closeText'];
```
Comment (wesleycho) [inline src/datepicker/datepicker.js]: Fix the code style of these two lines as well.

**Comment by Foxandxss** [comment]
I am not sure if this is something we should do.

I am open for a discussion about supporting i18n in our library in a new issue.

Thanks.

**Pull request closed without merging**